### PR TITLE
PFNano for native NanoV11

### PR DESCRIPTION
**Pull request by colizz**
Update the branch to prepare PFNano for native NanoV11, used by the first Run 3 SF campaign to calibrate DeepJet from native NanoV11.

This is a PR to merge the [`13_0_7_from124MiniAOD`](https://github.com/cms-jet/PFNano/tree/13_0_7_from124MiniAOD) branch to [`12_6_0`](https://github.com/cms-jet/PFNano/tree/12_6_0), with an additional commit (https://github.com/cms-jet/PFNano/commit/0dd9ffce12921f67bf81e8072734234697a85c29) to switch back several configurations.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,14 +17,10 @@ This format can be used with [fastjet](http://fastjet.fr) directly.
 For 2022 data and MC **NanoAOD v11** according to the [XPOG](https://gitlab.cern.ch/cms-nanoAOD/nanoaod-doc/-/wikis/Releases/NanoAODv11) and [PPD](https://twiki.cern.ch/twiki/bin/view/CMS/PdmVRun3Analysis) recommendations:
 
 ```
-cmsrel CMSSW_12_6_0_patch1
-cd CMSSW_12_6_0_patch1/src
+cmsrel CMSSW_12_6_5
```
Comment (demuller) [inline README.md:20]: any reason why switching to 12_6_5 is needed?

File: README.md [hunk 2]
```diff
@@ -62,48 +62,39 @@ Two imporant parameters that one needs to verify in the central nanoAOD document
...
---customise_commands="process.add_(cms.Service('InitRootHandlers', EnableIMT = cms.untracked.bool(False)));process.MessageLogger.cerr.FwkReport.reportEvery=1000;process.NANOAODoutput.fakeNameForCrab = cms.untracked.bool(True)" --nThreads 4 \
--n -1 --filein "/store/data/Run2022C/DoubleMuon/MINIAOD/10Dec2022-v1/2820000/dea1757f-d2ef-467a-9062-714775d00e45.root" --fileout file:nano_data2022ABCD.root \
---customise="PhysicsTools/PFNano/pfnano_cff.PFnano_customizeData_add_DeepJet"  --no_exec
-```
-<br>
-```
-cmsDriver.py nano_data_2022E --data --eventcontent NANOAOD --datatier NANOAOD --step NANO \
---conditions 124X_dataRun3_v14   --era Run3,run3_nanoAOD_124 \
+cmsDriver.py nano_data_2022CDE --data --eventcontent NANOAOD --datatier NANOAOD --step NANO \
+--conditions 124X_dataRun3_v15   --era Run3,run3_nanoAOD_124 \
```
Comment (demuller) [inline README.md:70]: Are those really the GTs for 2022 Partial ReReco Data? I find some contradicting information: [nanoAOD v11 wiki](https://gitlab.cern.ch/cms-nanoAOD/nanoaod-doc/-/wikis/Releases/NanoAODv11) vs [PdmV](https://twiki.cern.ch/twiki/bin/viewauth/CMS/PdmVRun3Analysis#Early_Analyses_Information_AN1), maybe we should double check with PPD?

File: python/pfnano_cff.py [hunk 3]
```diff
@@ -4,42 +4,20 @@
 from PhysicsTools.NanoAOD.common_cff import Var
 
 # keepInputs can take DeepCSV, DeepJet and DDX (any combination, or use empty placeholder list if no inputs are required)
+# from 13_0_X onwards, the default changes to:
```
Comment (demuller) [inline python/pfnano_cff.py:7]: already the case for 12_6_X

File: README.md (hunk 1 above)
Comment (colizz) [reply README.md:20]: here followed the convention that Z in X_Y_Z are always updated to the latest

File: README.md (hunk 2 above)
Comment (colizz) [reply README.md:70]: Good catch! I think the PdmV link is the latest one. From https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/124X_dataRun3_PromptAnalysis_v2 
it indicates "PromptAnalysis GT copied from 124X_dataRun3_Prompt_v10 with updated JEC/JR for 2022FG".

File: test/nano_data_2022FG_noPF_NANO.py
[the remark this answers is not in the document]
Comment (colizz) [reply test/nano_data_2022FG_noPF_NANO.py:1]: which yaml file sorry?

File: README.md (hunk 2 above)
Comment (demuller) [reply README.md:70]: Hm, yeah but you are referring to the Prompt GT, my question is about the ReReco GT: the first link I mentioned lists v14, while PdmV lists v15, and I am not sure if v15 is already the GT for the partial ReReco or if v15 will be the GT to be used for ReReco v2?

File: test/nano_data_2022FG_noPF_NANO.py
[the remark this answers is not in the document]
Comment (demuller) [reply test/nano_data_2022FG_noPF_NANO.py:1]: e.g. [here](https://github.com/cms-jet/PFNano/blob/12_6_0/test/card_example_dataFG.yml#L16), I am referring to the yaml files in the test folder

File: README.md (hunk 2 above)
Comment (colizz) [reply README.md:70]: ah, yes! By partial ReReco do you main the previous data with [10Dec2022] tag, and ReReco v2 refers to [27Jun2023]? If this is the case, we should already move to 27Jun2023 (both analysis, our SF derivation etc)

File: python/pfnano_cff.py (hunk 3 above)
Comment (colizz) [reply python/pfnano_cff.py:7]: changed to 12_6_X in new commit

File: test/nano_data_2022FG_noPF_NANO.py
[the remark this answers is not in the document]
Comment (colizz) [reply test/nano_data_2022FG_noPF_NANO.py:1]: The yaml files are also updated

File: README.md (hunk 2 above)
Comment (demuller) [reply README.md:70]: Ok, I might have confused myself concerning the meaning of Partial ReReco: Partial ReReco seems to be the one with the June2023 tag, see slide 7 of [PPD news](https://indico.cern.ch/event/1312617/contributions/5521520/attachments/2694090/4675543/23-08-03_News_PPD.pdf) and the corresponding [pMp link](https://cms-pdmv-prod.web.cern.ch/pmp/historical?r=27Jun2023&showDoneRequestsList=true). So I conclude that the GTs in your PR are the correct ones

**Review by demuller — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (colizz) [reply README.md:70]: ah I see. Thanks!

**Pull request merged**